src/UI/Component/Panel/Listing/Appointement.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Panel\Listing;
+
+/**
+ * Interface Appointment
+ * @package ILIAS\UI\Component\Panel\Listing
+ */
+interface Appointment extends \ILIAS\UI\Component\Component {
```

Comment:
Where does this "name deconstruction" take place? Not in the documentation view of the the ILIAS administration. Personally I would prefer /Panel/Listing/AppointmentListing but this also relates to the namespace discussion.